tools/cdk-build-tools/config/markdownlint.json

```diff
@@ -0,0 +1,30 @@
+{
+  "default": false,
+  "MD001": true,
```

Comment:
can we use the rule name instead of the code? 
Tells us which rule this is without having to reference somewhere, at least a first approximation.

Reply:
We can yeah. I had initially decided against because rules have *multiple* names... But I guess you're right this might be better In the long run. If only JSON supported comments... -_-